.github/workflows/coding.yml

```
@@ -0,0 +1,55 @@
+  
+  # PROGRAMMING LANGUAGE
+  # SCALA
+  build-scala-latest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: scala-latest
+      run: docker build . --file ./scala/build/Dockerfile.latest --tag "scala-latest:$(date +%s)" 
+  
+  # TEST
+  test-scala-coding:  
```

yamllint

[error] trailing spaces

(trailing-spaces)